src/Sulu/Bundle/SnippetBundle/Controller/SnippettypesController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SnippetBundle\Controller;
+
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use Sulu\Component\Content\Compat\Structure;
+use Sulu\Component\Content\Compat\StructureManagerInterface;
+use Sulu\Component\Rest\RequestParametersTrait;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Handles snippet types and defaults.
+ */
+class SnippettypesController extends Controller implements ClassResourceInterface
```

Comment:
Why did you change the name? It doesn't fit our conventions. To correct the resulting route (I guess that's what you have intended with that) we could also use the `Route` annotation, as we are doing in some other places.